portfolio-project-matching-app/components/Input.module.css

```diff
@@ -0,0 +1,11 @@
+.text {
+    @apply m-2 p-2 bg-gray-200 appearance-none border-2 border-gray-200 rounded-md text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-purple-500
+}
+
+.password {
+    @apply m-2 p-2 bg-gray-200 appearance-none border-2 border-gray-200 rounded-md text-gray-700 leading-tight focus:outline-none focus:bg-white focus:border-purple-500
```

Comment:
I like the `focus:bg-white`, it looks pretty dang slick.

Also, curious where you snagged the insight to use `leading-tight`, here. I don't think I would have known to reach for that.